jobs/ci_open_jenkins/build/cgt.groovy

```diff
@@ -15,5 +15,10 @@ new SbtMicroserviceJobBuilder('capital-gains-calculator').
         withSCoverage().
         build(this as DslFactory)
 
+new SbtMicroserviceJobBuilder('capital-gains-subscription').
+        withScalaStyle().
+        withSCoverage().
+        build(this as DslFactory)
+
 new BuildMonitorViewBuilder('CGT-MONITOR')
         .withJobs('capital-gains-calculator-frontend','capital-gains-calculator').build(this)
```

Comment:
Do we need an entry in here for the CGT-MONITOR?

Or do we need a new BuildMonitorViewBuilder?